src/patternfly/components/Table/table-grid.scss

```diff
@@ -139,7 +139,7 @@
   border: none;
 
   // reset cell modifications
-  tr:where(.#{$table}__tr) > * {
+  tr:where(.#{$table}__tr) > :where(th, td) {
```

Comment:
OK - and do we need specificity of 1 with `tr:where(.#{$table}__tr)` vs 0 for `:where(tr.#{$table}__tr)`?